django_cloud_deploy/workflow/__init__.py

```diff
@@ -188,24 +191,40 @@ def create_and_deploy_new_project(
             self._generate_section_header(
                 6, 'Static Content Serve Set Up (Take Up To 5 Minutes)',
                 self._TOTAL_NEW_STEPS))
-        self._statitc_content_workflow.serve_static_content(
+        self._static_content_workflow.serve_static_content(
             project_id, cloud_storage_bucket_name, static_content_dir)
 
-        print(
-            self._generate_section_header(
-                7, 'Create Service Account Necessary For Deployment',
-                self._TOTAL_NEW_STEPS))
-        secrets = self._generate_secrets(project_id, database_username,
-                                         database_password,
-                                         required_service_accounts)
+        if use_gke:
+            print(
+                self._generate_section_header(
+                    7, 'Create Service Account Necessary For Deployment',
+                    self._TOTAL_NEW_STEPS))
+            secrets = self._generate_secrets(project_id, database_username,
+                                             database_password,
+                                             required_service_accounts)
+
+            print(
+                self._generate_section_header(
+                    8, 'Deployment (Take Up To 20 Minutes)',
+                    self._TOTAL_NEW_STEPS))
+            admin_url = self._deploygke_workflow.deploy_new_app_sync(
+                project_id, cluster_name, django_directory_path,
+                django_project_name, image_name, secrets)
+        else:
+            print(
```

Comment:
Step 7 "Create Service Account Necessary For Deployment" is printed, but actually nothing is done. Maybe we should say this step is skipped when deploying on app engine.